transaction-status/src/lib.rs

```diff
@@ -558,6 +562,44 @@ impl EncodedTransaction {
     }
 }
 
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub struct LegacyTransactionByAddrInfo {
```

Comment:
nit: let's keep this and `impl From<LegacyTransactionByAddrInfo> for TransactionByAddrInfo` in storage-bigtable, since that's the only place they're used.